Donation-platform/src/main/java/com/google/DbCalls.java

```diff
@@ -141,6 +142,7 @@ public Tag getTagByName(String name) throws Exception {
     ArrayList<Tag> tagDataStore = new ArrayList<Tag>();
     if (categories != null) {
       for (Key key : categories) {
+        System.out.println(key);
```

Comment:
Does this need to be printed for debugging reasons?

Reply:
done